dashboard/test/ui/features/platform/header.feature

```diff
@@ -35,6 +35,7 @@ Feature: Header navigation bar
     And element "#header-help" is not visible
     And element "#header-about" is not visible
     And element "#header-incubator" is not visible
+    Then I change the browser window size to 1280 by 1024
```

Comment:
I had some issues locally testing where the window size wasn't returned to normal and the other tests failed because, now, menu items and such were off the screen. Does not seem to be an issue with how SauceLabs does things?? or maybe it's just lucky ordering.